Makefile

```diff
@@ -64,3 +64,13 @@ test: build/xdebug-filter.php
 docs:
 	cd docs && sphinx-build -W -b html -d _build/doctrees . _build/html
 .PHONY: docs
+
+yarn-install:
+	@docker run --rm -it -v ${PWD}:/usr/src/app -w /usr/src/app node:11-alpine yarn install
+	$(MAKE) yarn-build
```

Comment:
I think we should remove this if we're going to make our make API similar to yarn's